extension/src/paymentHandler/sessions-request.handler.js

```diff
@@ -3,26 +3,29 @@ import {
   createSetCustomFieldAction,
 } from './payment-utils.js'
 import c from '../config/constants.js'
-import { getPaymentMethods } from '../service/web-component-service.js'
+import { createSessionRequest } from '../service/web-component-service.js'
 
 async function execute(paymentObject) {
-  const getPaymentMethodsRequestObj = JSON.parse(
-    paymentObject.custom.fields.getPaymentMethodsRequest
+  const amountUpdatesRequestObj = JSON.parse(
```

Comment:
Why is it named as `amountUpdatesRequestObj` ?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1050/commits/0d6fb80dacde447429ffa26a8514864374db90ce#diff-a6ebee5cac76979c3469c418f46614bf4d3f8de5a426108afd27c1b292e55a80